ghost/core/core/frontend/helpers/ghost_head.js

```diff
@@ -47,9 +48,24 @@ function getMembersHelper(data, frontendKey) {
     if (!settingsCache.get('members_enabled')) {
         return '';
     }
+    const {scriptUrl} = getFrontendAppConfig('portal');
+
+    const colorString = (_.has(data, 'site._preview') && data.site.accent_color) ? data.site.accent_color : '';
+    const attributes = {
+        ghost: urlUtils.getSiteUrl(),
+        key: frontendKey,
+        api: urlUtils.urlFor('api', {type: 'content'}, true)
+    };
+    if (colorString) {
+        attributes['accent-color'] = colorString;
+    }
+    const dataAttributes = getDataAttributes(attributes);
+
+    if (colorString) {
```

Comment:
Was this leftover from something? `dataAttributes` is a string at this point 🤔

Reply:
Ugh yea thanks, it was exactly that, was supposed to be deleted! Fixed